bears/python/BanditBear.py

```diff
@@ -7,7 +8,8 @@
 from coalib.settings.Setting import typed_list
 
 
-@linter(executable='bandit')
+@linter(executable='bandit',
+        use_stderr=True)
```

Comment:
I understand the default value of ```use_stdout``` is ```True```. Not sure here, but maybe explicitly adding ```use_stdout=True``` makes the code easier to understand? (ref: http://api.coala.io/en/latest/Developers/Linter_Bears_Advanced.html)

Reply:
It doesn't matter as `use_stdout` is True by default. If I add it here, it will be redundant. And What's the point of adding one extra line. To understand this, one should have understanding of `linter` class and if anyone understand `linter` class, then IMO he should also know about default `use_stdout`.